.github/workflows/create_release.yml

```diff
@@ -0,0 +1,58 @@
+name: Create GitHub Release for Extension Package
+
+on:
+    workflow_dispatch:
```

Comment:
One thing we could do here, which is currently hard-coded [below](https://github.com/salesforce/salesforcedx-vscode-mobile/pull/32/files#diff-20b4af9a743edb3be348e23e792bf3fb0ad92637b44fafa6c5de878558abce7dR40), is give an input field to the workflow, that would allow the initiating user to create the description associated with the GitHub Release. Could be a nice way to make the "release notes" applicable to each release. Easy to do.